### ISSUE TITLE
Parasol.weightedSum()

### ISSUE THREAD

**Issue by wraseman** [joshhjacobson/parasol#21]
Hey Josh, 

Since we are implementing the weighted sum method, the name should be ```weightedSum()``` rather than ```weightedSums()``` for the Parasol API, in my opinion. Can you make this change?

Thanks!

**Comment by wraseman** [comment]
And the variable name should be 'weighted sum' rather than 'aggregate score'. That'll probably be more intuitive for the user.

**Comment by joshhjacobson** [comment]
Hi Billy,

At the time of writing, my thinking was that since we are computing a weighted sum for each solution, the method overall creates a set of weighted sums for all solutions. Now though, I see your point, and also we want to keep the language consistent with other methods, e.g. `cluster()`. However, while you would say "cluster the data," you probably wouldn't say "weighted sum the data." For comparison, there is also "compute the clusters" and "compute the weighted sums" ... 

In short, I suppose the nomenclature is weird either way, so I'm happy to make the change if that's what you want to do.

**Comment by wraseman** [comment]
Let's go with ```weightedSum()```. I think of it as "apply the weighted sum method" to the data. It just seems more natural to me. Although, I agree that if we were to *really* get into it, our language could probably be more consistent overall but I'm sure we'll always feel that way.
